src/logic/__tests__/linkGithubIdentityTest.js

```diff
@@ -1,5 +1,12 @@
-import linkGithubIdentity from '../linkGithubIdentity';
+import {
+  credential as createCredential,
+  credentialInUseError,
+  firebaseError,
+  githubProfile as createGithubProfile,
```

Comment:
From a naming standpoint I prefer `githubProfileFactory` etc. rather than `createGithubProfile`, since the things we are importing aren’t themselves functions. This is the convention I am using [in the CodeMirror pull](https://github.com/popcodeorg/popcode/pull/1670/files#diff-48da82123a3b32cc381d1f03147a30a6R9)